src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/Primitives.cs

```diff
@@ -173,7 +173,7 @@ internal enum CertContextPropId : int
         // CERT_DELETE_KEYSET_PROP_ID is not defined by Windows. It's a custom property set by the framework
```

Comment:
It is now, it's called CERT_CLR_DELETE_KEY_PROP_ID in wincrypt.h :) (125)

Reply:
do we care about renaming it?

Reply:
done it to avoid confusion in the future